### ISSUE TITLE
Yes-No data for M2M and S2S ICM Team and Supervisor Monitoring Data not showing on the dashboard

### ISSUE THREAD

**Issue by danieltomabba** [xlg8/APMIS-Project#583]
@sundayAdu In creating dashboard categories for M2M and S2S modalities, we observed that the dashboard elements are not being displayed for yes-no questions. This is true for referenceValue= true showing the percentage of yes for each question. 

The actual output is Yes/(Yes+No) for each region, province or district output. 

See diagram definition used

```
[

{"fieldId":"residentOfCluster","formId": "ICMSupervisorM2M-readonly", "referenceValue": true, "caption":"Is supervisor resident of assigned cluster?", "stack":"ResidentAssignedCluster"},

  {"fieldId":"trainedBeforeCampaign","formId": "ICMSupervisorM2M-readonly", "referenceValue": true, "caption":"Has supervisor been trained before this campaign?", "stack":"Trained"}

]
```\

and example form definition used

```
[
  {
    "type": "label",
    "id": "ICMSupervisorM2M-readonly",
    "caption": "<h2>Supervisor Monitoring Form M2M</h2>"
  },
  
  
  {
	"type": "section",
	"id": "ClusterInformation"
	},{
	"type": "label",
	"id": "ClusterInfoLabel-readonly",
	"caption": "<h3>Cluster Information:</h3>"
	},{
	"type": "date",
	"id": "VisitDate",
	"caption": "Visit Date",
	"important": "true"
	},{
	"type": "text",
	"id": "supervisorName",
	"caption": "Supervisor Name",
	"important": "true"
	},{
	"type": "text",
	"id": "monitorName",
	"caption": "Monitor Name",
	"important": "true"
	},
	{
	"type": "section",
	"id": "SupInformation"
	},{
	"type": "label",
	"id": "SupervisorINfoLabel-readonly",
	"caption": "<h2>Supervisor Information:</h2>"
	},{
	"type": "yes-no",
	"id": "residentOfCluster",
	"caption": "1- Is supervisor resident of assigned cluster?",
	"important": "true"
	},{
	"type": "yes-no",
	"id": "trainedBeforeCampaign",
	"caption": "2- Has supervisor been trained before this campaign?",
	"important": "true"
	},{
	"type": "yes-no",
	"id": "carryingOPV",
	"caption": "3- Is supervisor carrying OPV (and Vitamin A or Albendazole, if appropriate)?",
	"important": "true"
	},{
	"type": "yes-no",
	"id": "revisedMaps",
	"caption": "4- Is supervisor carrying maps with clearly marked team areas & boundaries?",
	"important": "true"
	},{
	"type": "yes-no",
	"id": "fillingHouseholdsCheckList",
	"caption": "5- Is supervisor carrying and filling households check list?",
	"important": "true"
	},{
	"type": "yes-no",
	"id": "TransportSupervision",
	"caption": "6- Is transport provided for supervision based on micro-plan?",
	"important": "true"
	},{
	"type": "dropdown",
	"id": "TransportTypeSupervision",
	"caption": "7- Does supervisor use transport in the field? Please select",
	"important": "true",
	"options":[{"key":"NoTransport","caption":"No Transport","order":"1"},
		{"key":"Car","caption":"Car","order":"2"},
		{"key":"Motorbike","caption":"Motorbike","order":"3"},
		{"key":"Animal","caption":"Animal","order":"4"}]
	},
{
	"type": "section",
	"id": "SourceSection"
	},{
	"type": "label",
	"id": "datasourceLabel-readonly",
	"caption": "<h3>Form Source</h3>"
	},{
	"type": "label",
	"id": "readonly-label",
	"caption": "File name: ICM Checklist (H2H).xlsx | Worksheet: Supervisor | Date provided: May 15th 2022" 
	}

]

```

**Comment by danieltomabba** [comment]
@sundayAdu Please can you verify the right syntax for implementing the numerator and denominator for Yes-No and the total for testing this? Thanks

**Comment by danieltomabba** [comment]
Before I can close this issue, we need the wiki updated with the right syntax to change the diagram definitions before we can test this to close it. @sundayAdu Please respond accordingly. I sent this request last month so we could test it out and close it. Thank you very much @RaziaMahmodi @bnygren

**Comment by sundayAdu** [comment]
resolved... https://github.com/AFG-Polio-Data/APMIS-Project/wiki/Creating-Diagram-Definitions-for-Campaigns#counting-yesno-in-diagrams

**Comment by bnygren** [comment]
Not tested, but will close until otherwise noted as needing review. The current dashboard will likely be superseded by improved visualization tools.
